examples/website-demos/social-messaging/src/App.tsx

```diff
@@ -1,17 +1,106 @@
+import { useEffect, useState } from 'react';
+
+import { ChannelSort, LiteralStringForUnion, StreamChat } from 'stream-chat';
+import { Channel, ChannelList, Chat } from 'stream-chat-react';
+
+import 'stream-chat-react/dist/css/index.css';
 import './styles/App.scss';
 
-import { ChannelListContainer } from './components/ChannelList/ChannelListContainer';
 import { ChannelContainer } from './components/ChannelContainer/ChannelContainer';
+import { SocialChannelList } from './components/SocialChannelList/SocialChannelList';
+import { SocialChannelListFooter } from './components/SocialChannelList/SocialChannelListFooter';
+import { SocialChannelListHeader } from './components/SocialChannelList/SocialChannelListHeader';
+import { SocialChannelPreview } from './components/ChannelPreview/SocialChannelPreview';
 
 import { useTheme } from './hooks/useTheme';
+import { SideDrawer } from './components/SideDrawer/SideDrawer';
+import { SocialMessageInput } from './components/MessageInput/SocialMessageInput';
+
+const apiKey = process.env.REACT_APP_STREAM_KEY;
+const user = process.env.REACT_APP_USER_ID;
+const userToken = process.env.REACT_APP_USER_TOKEN;
+
+const userToConnect: { id: string; name?: string; image?: string } = {
+  id: user!,
+  name: user!,
+  image: 'https://ca.slack-edge.com/T02RM6X6B-U01J8HMLA4F-d7bab110afb4-512',
+};
+
+const filters = { type: 'messaging', members: { $in: [user!] } };
+
+const options = { state: true, watch: true, presence: true, limit: 8 };
```

Comment:
some of these default to true, not sure which ones tbh, so would be nice if you could find out then we wouldn't need to include

Reply:
https://getstream.io/chat/docs/javascript/query_channels/
Looks like true is the default for state and watch, presence doesn't appear to be a thing, so I removed.